core/iwasm/interpreter/wasm_mini_loader.c

```diff
@@ -5265,7 +5306,41 @@ wasm_loader_check_br(WASMLoaderContext *loader_ctx, uint32 depth,
 #endif
             PUSH_TYPE(types[i]);
         }
-        return true;
+
+#if WASM_ENABLE_FAST_INTERP != 0
+        emit_br_info(target_block, opcode == WASM_OP_BR);
+#endif
+
+        /* Restore the stack data, note that frame_ref_bottom,
+           frame_reftype_map_bottom, frame_offset_bottom may be
```

Comment:
Maybe remove `frame_reftype_map_bottom` in the comment since the corresponding data member was not in the 1.3.x branch